modules/dedicated-cluster-admin-enable.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// administering_a_cluster/cluster-admin-role.adoc
+
+[id="dedicated-cluster-admin-enable"]
+= Enabling the cluster-admin role for your cluster
+
+The cluster-admin role must be enabled at the cluster level before it can be assigned to a user.
+
+.Prerequisites
+. Submit a ticket to Service Reliability Engineering (SRE) to enable `cluster-admin` for your cluster. 
```

Comment:
@sheriff-rh @jewzaam @lamek The customer will not know how to contact SRE, and in fact doesn't have a way to do so directly. I believe this should instead say something like, "Open a technical support case with Red Hat to request that `cluster-admin` be enabled for your cluster."